main.c

```diff
@@ -17,6 +17,7 @@
 #include <solv/solv_xfopen.h>
 
 G_DEFINE_AUTOPTR_CLEANUP_FUNC(Pool, pool_free);
+G_DEFINE_AUTOPTR_CLEANUP_FUNC(Queue, queue_free);
```

Comment:
This is not being used and is in fact wrong, since queue_free does not free the pointer itself.